net/portmapper/upnp.go

```diff
@@ -321,5 +323,7 @@ func parseUPnPDiscoResponse(body []byte) (uPnPDiscoResponse, error) {
 		return r, err
 	}
 	r.Location = res.Header.Get("Location")
+	r.Server = res.Header.Get("Server")
+	r.USN = res.Header.Get("USN")
```

Comment:
technically `Get("Usn")` saves an alloc here and is equivalent 😛

Reply:
eh how come changing the caps removes an alloc?